### PR TITLE
dm/tag-after-branch

### PR DESCRIPTION
**Pull request by dmerejkowsky**

### CODE REVIEW

File: tbump/git_bumper.py [hunk 1]
```diff
@@ -95,7 +95,8 @@ def compute_commands(self, new_version):
         tag_message = tag_name
         res.append(["tag", "--annotate", "--message", tag_message, tag_name])
         if self.remote_branch:
-            res.append(["push", self.remote_name, self.remote_branch, tag_name])
+            res.append(["push", self.remote_name, self.remote_branch])
+            res.append(["push", self.remote_name, tag_name])
```
Comment (tux3) [inline tbump/git_bumper.py]: That should work! But maybe what you want is the `--atomic` flag of git push?

Comment (dmerejkowsky) [reply tbump/git_bumper.py]: Let's keep it as two commands just in case (man page says that atomic pushes requires stuff from the server)

Comment (tux3) [reply tbump/git_bumper.py]: Sure, sounds fine!

File: tbump/test/test_main.py [hunk 2]
```diff
@@ -30,9 +30,15 @@ def check_local_git_status(test_repo):
     assert "v1.2.41-alpha-2" in out
 
 
-def has_pushed(test_repo, pushed=True):
-    tbump.git.run_git(test_repo, "ls-remote", "origin", "refs/tags/v1.2.41-alpha-2")
+def hash_pushed_tag(test_repo):
```
Comment (maximerety) [inline tbump/test/test_main.py]: @dmerejkowsky `has` inadvertently became `hash`, also in `hash_pushed_branch`

Comment (dmerejkowsky) [reply tbump/test/test_main.py]: Too much crypto :)

**Pull request merged**